### DISCUSSION TITLE
Personal/One time membership - "personal use" for instances - is there a limit?

### DISCUSSION THREAD

**Discussion by accessiblepixel** [photoprism/photoprism#3385] [Q&A]
Hi.

Hope the day finds you well.

I'm looking at doing a one time donation to get access to different themes and all the features in photoprism (at this moment in time I could not afford a regular commitment, due to health/personal issues) so I'm looking at the $25 option on github

My question is on the pricing/versions compared, there's the column that says "Instances: personal use" and I'm wondering if that means if there's a limitation on how many instances can be registered under a donation.

There's only a couple of us in my family, but we all have media collections that are very different. Two of us would probably use two instances hosted at home, and then a 3rd family member would probably have an instance somewhere web accessible... but I'm not sure if that "personal use" column means there's a restriction on how an instance is validated as "supported" or not.

I'm not particularly looking for support, the only things I'll ever report are bugs so technical support isn't something I need or require since it seems the docs look pretty good...

But I guess it's two questions, is a one time personal license enough to cover multiple personal instances and is that the right amount to make for what I require for a couple of instances for family members?

Thank you in advanced - I did search issues and discussions to see if there was anything related to what I'm asking but didn't find anything explaining (mainly) the "personal use" for instances. Sorry if any of my questions are dumb :)

Kind regards

**Comment by ippocratis** [comment]
Get a sponsorship
They start from 8/month
Then cancel
You will only get charged once
Thats what I did
But idc for extra features themes etc
Just want to say thank you once in a while

**Comment by graciousgrey** [comment]
Hi 👋 

> I'm looking at doing a one time donation to get access to different themes and all the features in photoprism (at this moment in time I could not afford a regular commitment, due to health/personal issues) so I'm looking at the $25 option on github

The $25 tier gives you access to [essentials features](https://www.photoprism.app/editions#compare) for 12 months.

Information on lifetime options and possible discounts can be found [here](https://www.photoprism.app/kb/membership).

>My question is on the pricing/versions compared, there's the column that says "Instances: personal use" and I'm wondering if that means if there's a limitation on how many instances can be registered under a donation.

We currently have not specified a hard limit because it depends on the context:

(a) If you are a family with 10 happy kids, then 10 instances is personal use.

(b) If you're a freelance photographer who has additional instances for private images and his/her partner, then that's personal use.

(c) If you are a business or government agency, even one instance does not count as personal use.

(d) If you share your account on Reddit with other people, that is not personal use.

**Reply by lastzero** [reply]
We've added your question to our Membership FAQ so others can easily find the answer:

https://www.photoprism.app/kb/membership#is-there-a-limit-for-personal-use

**Comment by accessiblepixel** [comment]
OK... I'm still incredibly confused by it all. I want to support open source software but all this reminds me of the old expression "clear as mud".

I looked on the page linked about lifetime membership but the link was just an email address? From what I could decipher from that page 24 months of 'essential' is then a lifetime?

How does license validation work?

**Reply by lastzero** [reply]
While we finish our Stripe integration (this will take a few more days), the easiest way to get a lifetime Plus membership is to sign up for the Gold tier on Patreon and cancel after a year. If you pay annually, you'll get an additional discount.

Alternatively, you can sign up for Silver and get a lifetime Plus membership after 2 years.

Note that you are not purchasing a license, as much of our source code is available in the [public GitHub repository](https://github.com/photoprism/photoprism), with the user management web UI as the only exception. It is in a private repository currently only accessible to our members, see our [Membership FAQ](https://www.photoprism.app/kb/membership).

**Reply by lastzero** [reply]
We plan to roll out Stripe later today, once all tests have been successfully completed. It will be a new payment option so you don't have to link external accounts, e.g. Patreon.

This is the main reason we have been developing our own membership portal over the past few months. If you haven't followed our team closely, I can imagine that this may not have become entirely clear. Should you be interested in our daily work, one option is to join the private members chat to connect with us and other users.
